### DISCUSSION TITLE
how i can connect only to one dns over tls server inside Config file?

### DISCUSSION THREAD

**Discussion by osevan** [DNSCrypt/dnscrypt-proxy#2253] [Q&A] [answered]
How can i connect with direct url to Servers inside Config file. 

Attention : no servername i mean Direct url for dns over tls url inside config.

And disabling dns probing for fastest servers. 

Thanks and

Best regards

**Comment by jedisct1** [comment, marked as answer]
`dnscrypt-proxy` doesn't support DNS over TLS.

**Reply by osevan** [reply]
And with doh how  i can do this?

**Reply by jedisct1** [reply]
Isn't that covered in the [documentation](https://github.com/DNSCrypt/dnscrypt-proxy/wiki) already?
